pkg/workloads/cortex/client/setup.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from setuptools import setup, find_packages
+from setuptools.command.install import install
+
+import sys
+
+
+class InstallBinary(install):
+    def run(self):
+        install.run(self)
+
+        import requests
+        from pathlib import Path
+        import sys
+        import os
+        import stat
+
+        dest_path = os.path.join(self.install_lib, "cortex", "binary", "cli")
+
+        if not os.path.exists(dest_path):
+            platform = sys.platform
+
+            if platform != "darwin" and platform != "linux":
+                raise Exception("cortex is only on mac and linux")
+
+            cortex_version = self.config_vars["dist_version"]
+
+            if "dev" in cortex_version:
+                cortex_version = "master"
+
+            download_url = f"https://s3-us-west-2.amazonaws.com/get-cortex/{cortex_version}/cli/{platform}/cortex"
+
+            with open(dest_path, "wb") as f:
```

Comment:
nit: if this is a clean swap, I'd go for it: https://stackoverflow.com/a/39217788